### DISCUSSION TITLE
Shortcuts to Converter, Date calculator, and TIme Zones

### DISCUSSION THREAD

**Discussion by victor-falcon** [sadellie/unitto#158] [Ideas]
There some launchers, like Niagara, that allow you to search through all your application shortcuts and launch them quickly.

The problem is that Unitto has no shortcut, but I will like to be able to search for Converter, for example, and launch Unitto in that view directly.

**Comment by sadellie** [comment]
Already implemented.

# Dynamic shortcuts
Latest tools that users have used. This list is automatically updated every time user switches between different tabs.

<img src="https://github.com/sadellie/unitto/assets/10384356/775132be-12c4-4e4b-9326-27c4a1154441" width="50%" />

# PInned shortcuts
**Settings** -> **Starting screen** -> **Icon button on the right**

<img src="https://github.com/sadellie/unitto/assets/10384356/c22daa51-b640-48e6-b880-06ce068f7213" width="50%" />

**Reply by victor-falcon** [reply]
why I don't have this? I have the latest version here on GitHub? 🤔

**Reply by victor-falcon** [reply]
![image](https://github.com/sadellie/unitto/assets/238766/8d320e9a-028c-4721-8061-978452118fb7) 
![image](https://github.com/sadellie/unitto/assets/238766/43ae9265-f4d1-4d10-a3b1-7085fef645ec) 
![image](https://github.com/sadellie/unitto/assets/238766/87877b17-6563-4cbb-a9b2-cba3d371ec33)

**Reply by sadellie** [reply]
fixed in #161
